CONTRIBUTING.md

```diff
@@ -74,6 +75,20 @@ A good bug report should be as detailed as possible, so that others won't have t
 
 **[Request a feature](https://github.com/videojs/video.js/issues/new)**
 
+### Labels
+
+There are a few labels that might be added to your issue or PR by a maintainer. Here's a quick rundown of what they mean:
+
+| Label | Issue or PR | Description |
+|-------|-------------|-------------|
+| confirmed | Issue and PR | Issue: marks as reproducible by a maintainer. PR: marked by a maintainer as ready to be merged|
+| 5.x | PR | Marks as a change to the 5.x branch only |
+| bug | Issue | Marks as a confirmed bug by a maintainer |
+| first-timers-only | Issue | Marked as a good bug or enhancement for first time contributors only |
```

Comment:
@gkatsev what's the difference between `first-timers-only` and  `good first issue`? I thought `good first issue` was what you were flagging "a good bug or enhancement for first time contributors", but I'm not entirely clear why an issue would be for "first timers **only**". Are you suggesting that people who aren't "first timers" **shouldn't** work on those issues? That seems like it's much less important that "a first timer is encouraged to work on this issue rather than on more complicated ones", so I'd expect to see `good first issue` described here rather than `first-timers-only`.

Reply:
Maybe this is something to elaborate on but `first-timers-only` should be super simple issues that are strongly encouraged to be done by people who are coming into open source for the first time (or well, recently) and well-established people should try and not address those issues. `good first issue` can be directed at anyone as a good point to get into the project.
This was started by Kent C Dodds and he has a blog post about it: https://blog.kentcdodds.com/first-timers-only-78281ea47455
so far, I think using it has turned out pretty well.

Reply:
thanks for the question @OwenEdwards! I've updated both guides to be more clear on the meanings of these two labels.